<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoopWorkGUIClass</class>
 <widget class="QWidget" name="CoopWorkGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoopWorkGUI</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_open">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_filepath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>841</width>
     <height>631</height>
    </rect>
   </property>
   <widget class="QWidget" name="wdt_scroll" native="true">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>69</y>
      <width>641</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Intersect</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CoopWorkGUI.qrc"/>
 </resources>
 <connections/>
</ui>
